service.php

```diff
@@ -86,7 +86,7 @@ function page_optimize_service_request() {
 
 function page_optimize_build_output() {
 	global $page_optimize_types;
-	ob_start( 'ob_gzhandler' );
+	ob_start();
```

Comment:
I think we can just skip output buffering and return the content at the end like:
```php
'content' => $pre_output . $output,
```

Reply:
Right! No need for buffering now that we don't `gzip`.